gradlew

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```

Comment:
I assume all changes in here were done by gradle itself.

Reply:
Yep.